paywall/src/__tests__/data-iframe/blockchainHandler/keyStatus.test.js

```diff
@@ -124,6 +124,17 @@ describe('key status manipulation', () => {
         )
       ).toBe('expired')
     })
+
+    it('should recognize a key with expiration of 0 but with transactions as valid', () => {
```

Comment:
I guess that is optimistic unlocking? If we have a pending transaction we assume the key is valid?

Reply:
That may be the case